src/components/DataTable/DataTable.tsx

```diff
@@ -435,25 +338,9 @@ class DataTable extends React.PureComponent<CombinedProps, DataTableState> {
     );
   };
 
-  private renderFooter = () => {
```

Comment:
Removing the absolute positioning of the first column cells made positioning the footer content seem egregious. I've changed the footer content to be rendered within a `div` as a sibling to table's scroll container so calculating/setting/updating the footer's height and the scroll container's bottom margin is no longer needed.